### ISSUE TITLE
Maps are broken

### ISSUE THREAD

**Issue by Orphansmith** [ben/foundry-ironsworn#573]
Saw the map for a moment. As soon as I dropped a character token onto the map it went black. Updated to today's update, and it's still all black. 3D dice won't accurately display on the map, all the tokens are gone, pretty much only the side chat bar works.

**Comment by ben** [comment]
This sounds like a foundry issue, or perhaps a graphics driver problem. Does everything work okay with, say, an dnd5e world? Could ask in the foundry discord for help as well.

**Comment by Orphansmith** [comment]
I'm running this in forge-vtt. When I start a different game world I'm able to see the map correctly. When I open Starforged I get a `DevTools failed to load source map: Could not load content for chrome-extension://blah-blah-blah/browser-polyfill.js.map: System error: net::ERR_FILE_NOT_FOUND.` message in the console.

**Comment by Orphansmith** [comment]
I also tried making a new map, but I get the same error. This time the system can't even render a thumbnail for the map.

**Comment by ben** [comment]
What kind of a computer are you running? Which browser are you using? Has Foundry/Forge worked for you for any other systems?

FWIW, that console error isn't worrying, it's a chrome extension that didn't give you a source map. No biggie.

**Comment by Orphansmith** [comment]
I'm running forge through a PC and the latest version of Chrome. Forge works for other systems, Ironsworn & Starforged is the only system giving me this issue. I also tried turning off all of my modules in case that was the problem, but it still persists.

There is another console error, `failed to execute texImage2D on WebGL2RenderingContext`. This error doesn't appear when loading any other system.

**Comment by ben** [comment]
Oooh that is odd. Can you copy/paste the entire contents of the console here?

**Comment by Orphansmith** [comment]
Sure,

`DevTools failed to load source map: Could not load content for chrome-extension://cfhdojbkjhnklbpkdaibdccddilifddb/browser-polyfill.js.map: System error: net::ERR_FILE_NOT_FOUND`

Then Foundry VTT basically loads everything. There's a few warnings saying that the avatar player images aren't found for chat portrait. But turning the chat portraits module off doesn't change anything. Finally I get,

```
Uncaught DOMException: Failed to execute 'texImage2D' on 'WebGL2RenderingContext': The image element contains cross-origin data, and may not be loaded.
    at r.upload (https://<me>.forge-vtt.com/scripts/pixi.min.js:8:109415)
    at r.upload (https://<me>.forge-vtt.com/scripts/pixi.min.js:8:113123)
    at e.updateTexture (https://<me>.forge-vtt.com/scripts/pixi.min.js:8:215193)
    at e.bind (https://<me>.forge-vtt.com/scripts/pixi.min.js:8:213523)
    at r.bindAndClearTexArray (https://<me>.forge-vtt.com/scripts/pixi.min.js:8:232029)
    at r.drawBatches (https://<me>.forge-vtt.com/scripts/pixi.min.js:8:232873)
    at r.flush (https://<me>.forge-vtt.com/scripts/pixi.min.js:8:233262)
    at r.stop (https://<me>.forge-vtt.com/scripts/pixi.min.js:8:233628)
    at t.setObjectRenderer (https://<me>.forge-vtt.com/scripts/pixi.min.js:8:146772)
    at t.flush (https://<me>.forge-vtt.com/scripts/pixi.min.js:8:146867)
r.upload @ core.min.mjs:8
r.upload @ core.min.mjs:8
e.updateTexture @ core.min.mjs:8
e.bind @ core.min.mjs:8
r.bindAndClearTexArray @ core.min.mjs:8
r.drawBatches @ core.min.mjs:8
r.flush @ core.min.mjs:8
r.stop @ core.min.mjs:8
t.setObjectRenderer @ core.min.mjs:8
t.flush @ core.min.mjs:8
e.renderAdvanced @ display.min.mjs:8
e.render @ display.min.mjs:8
render @ foundry.js:26362
e.render @ display.min.mjs:8
e.render @ display.min.mjs:8
e.render @ display.min.mjs:8
r.render @ core.min.mjs:8
e.render @ app.min.mjs:8
t.emit @ ticker.min.mjs:8
e.update @ ticker.min.mjs:8
_tick @ ticker.min.mjs:8
requestAnimationFrame (async)
_tick @ ticker.min.mjs:8
requestAnimationFrame (async)
_tick @ ticker.min.mjs:8
requestAnimationFrame (async)
_tick @ ticker.min.mjs:8
requestAnimationFrame (async)
_tick @ ticker.min.mjs:8
requestAnimationFrame (async)
_tick @ ticker.min.mjs:8
requestAnimationFrame (async)
e._requestIfNeeded @ ticker.min.mjs:8
e.start @ ticker.min.mjs:8
start @ ticker.min.mjs:8
e.init @ ticker.min.mjs:8
(anonymous) @ app.min.mjs:8
e @ app.min.mjs:8
#createApplication @ foundry.js:23356
initialize @ foundry.js:23258
initializeCanvas @ foundry.js:8094
await in initializeCanvas (async)
setupGame @ foundry.js:7929
await in setupGame (async)
_initializeGameView @ foundry.js:8999
_initializeView @ foundry.js:8975
initialize @ foundry.js:7825
await in initialize (async)
window.addEventListener.once @ foundry.js:79693
```

**Comment by ben** [comment]
Hmm. Very strange. Can you check by visiting [this server](https://starforged.straub.cc)? Use `Gamemaster` / `abc` for credentials. That CORS error is strange.

**Comment by ben** [comment]
Also the paths to the scripts look like `https://<me>.forge-vtt.com/..`, which is a weird URL pattern to be seeing in a backtrace. Or is that your own redaction?

**Comment by Orphansmith** [comment]
No issues on that test server. Maybe https://forge-vtt.com/ is at fault?
My own redaction.

**Comment by ben** [comment]
OK yeah, that could make sense. The clue is this line:

```
The image element contains cross-origin data, and may not be loaded.
```

I wonder what image it's referring to. Are you pulling a scene background from another source, or are you using the sector that comes with the system?

**Comment by Orphansmith** [comment]
I found the issue.

I deleted all of my maps and refreshed the page. I was able to create maps again no problem.

However, I'm using Lancer system icons for some of the characters (from forge-vtt's Bazaar). If I drag one of these to the map the error appears again. For that point on, even if I delete the map and make a new one, the system won't display maps until I delete all maps and refresh the page.

So it's probably an error with Forge-VTT where the system is trying to pull assets from two locations and has some kind of problem resolving that.
